src/Icon/Icon.js

```diff
@@ -22,9 +22,13 @@ const Icon = React.forwardRef(({ glyph, size, className, ...props }, ref) => {
         style = { fontSize: ICON_SIZES[size] };
     }
 
+    const Tag = isInButton ? 'i' : 'span';
```

Comment:
I was hesitant to use the `Icon` component because it uses a `span` instead of `i`. Should we go back to fundamental-styles so that all icons use an `i` as in nui-widgets?

Reply:
After the discussion this morning, I think they're in the process of moving all icons to their own `<i>` tag- https://github.com/SAP/fundamental-styles/pull/1604

We can probably just go ahead and use `i` tags since it doesn't cause any styling issues and then styles will catch up. Up to you though

Reply:
👍 If we could use the `Icon` component and use `i` tags to get ahead of it where possible that would save us time down the line.